container.fc

```diff
@@ -12,6 +12,8 @@
 /usr/local/s?bin/docker.*		--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/s?bin/containerd.*		--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/local/s?bin/containerd.*		--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
+/usr/bin/nerdctl		--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
```

Comment:
Remove these specifications and only label /var/lib/nerdctl below, and I will merge.